pkg/sdkserver/sdkserver.go

```diff
@@ -694,3 +775,36 @@ func (s *SDKServer) updatePlayerCapacity() error {
 	s.recorder.Event(gs, corev1.EventTypeNormal, "PlayerCapacity", fmt.Sprintf("Set to %d", gs.Status.Players.Capacity))
 	return nil
 }
+
+// updateConnectedPlayers updates the Player IDs and Count fields in the GameServer's Status.
+func (s *SDKServer) updateConnectedPlayers() error {
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return errors.New(string(runtime.FeaturePlayerTracking) + " not enabled")
+	}
+	s.logger.WithField("playerIDs", s.gsConnectedPlayers).Debug("updating connected players")
```

Comment:
Might go also after lock.
```suggestion
        s.gsUpdateMutex.RLock()
	s.logger.WithField("playerIDs", s.gsConnectedPlayers).Debug("updating connected players")
```

Reply:
Good catch!